Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1896,46 +1897,74 @@ extension StatusTableViewController: AddEditOverrideTableViewControllerDelegate
     }
 }
 
-extension StatusTableViewController: CGMManagerSetupViewControllerDelegate {
-    func cgmManagerSetupViewController(_ cgmManagerSetupViewController: CGMManagerSetupViewController,
-                                       didSetUpCGMManager cgmManager: CGMManagerUI)
-    {
+extension StatusTableViewController {
+    fileprivate func addCGMManager(withIdentifier identifier: String) {
+        switch setupCGMManager(withIdentifier: identifier) {
+        case .failure(let error):
+            log.default("Failure to setup CGM manager with identifier '%{public}@': %{public}@", identifier, String(describing: error))
+        case .userInteractionRequired(var setupViewController):
+            setupViewController.cgmManagerCreateDelegate = self
+            setupViewController.cgmManagerOnboardDelegate = self
+            setupViewController.completionDelegate = self
+            show(setupViewController, sender: self)
+        case .success:
+            log.default("CGM manager with identifier '%{public}@' created and onboarded", identifier)
+        }
+    }
+}
+
+extension StatusTableViewController: CGMManagerCreateDelegate {
+    func cgmManagerCreateNotifying(_ notifying: CGMManagerCreateNotifying, didCreateCGMManager cgmManager: CGMManagerUI) {
+        log.default("CGM manager with identifier '%{public}@' created", cgmManager.managerIdentifier)
         deviceManager.cgmManager = cgmManager
     }
 }
```

Comment:
[nit] Should the create delegate be the `deviceManager`?

Reply:
See previous comment.